requirements.txt

```diff
@@ -0,0 +1,2 @@
+mlflow
+elasticsearch_dsl=6.4.0
```

Comment:
Any reason to pin elasticsearch_dsl version ?

Reply:
It is the version compatible with the Elasticsearch cluster I have

Reply:
The latest version doesn't work ?

Reply:
No I tested it